nat_gateway/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "private_route_tables" {
```

Comment:
@MattiasGees can you elaborate more? How many route tables you want? I assume you are looking to have one route table for private subnet and one for public. If you have multiple AZs you can have one nat gateway per AZ and this is supported. More complex scenarios need a custom module in my opinion